ddtrace/llmobs/_llmobs.py

```diff
@@ -690,9 +716,11 @@ def _tag_span_tags(span: Span, span_tags: Dict[str, Any]) -> None:
             log.warning("span_tags must be a dictionary of string key - primitive value pairs.")
             return
         try:
-            current_tags = span.get_tag(TAGS)
-            if current_tags:
-                span_tags.update(json.loads(current_tags))
+            current_tags_str = span.get_tag(TAGS)
```

Comment:
Would we need a deprecation notice for this? Since it technically changes the behavior of `annotate` ... and maybe some users relied on tags _not_ being overriden

Reply:
might be worth putting in the `other` section in case anyone sees a change and is curious/confused